config.py

```diff
@@ -104,6 +104,9 @@ class DevelopmentConfig(Config):
     SURVEY_PASSWORD = os.getenv('SURVEY_PASSWORD', 'secret')
     SURVEY_AUTH = (SURVEY_USERNAME, SURVEY_PASSWORD)
 
+    TEMPLATES_AUTO_RELOAD = True
+    EXPLAIN_TEMPLATE_LOADING = True
```

Comment:
Do we want EXPLAIN_TEMPLATE_LOADING on by default? It does tend to flood logs.

Reply:
Hmm, good point. It can be useful if there are issues around template loading. I'll conduct a straw poll tomorrow and see what people prefer.

Reply:
Is `TEMPLATES_AUTO_RELOAD` needed? See https://github.com/pallets/flask/issues/1907 and https://github.com/pyvec/elsa/issues/44